src/theme.ts

```diff
@@ -466,35 +509,137 @@ export const newTheme = createTheme({
         },
       },
     },
+    MuiTypography: {
+      defaultProps: {
+        variantMapping: {
+          bodyMdBold: 'p',
+          bodyMdRegular: 'p',
+          bodySmBold: 'p',
+          bodySmRegular: 'p',
+          headingLg: 'h1',
+          headingMd: 'h2',
+          headingSm: 'h3',
+        },
+      },
+    },
   },
   elevation: {
     bottom: {
       big: {
-        light: '0px 4px 40px 0px #00000014',
-        medium: '0px 4px 40px 0px #0000001F',
+        light: '0rem 0.25rem 2.5rem 0rem #00000014',
+        medium: '0rem 0.25rem 2.5rem 0rem #0000001F',
       },
       small: {
-        light: '0px 4px 20px 0px #00000014',
-        medium: '0px 4px 20px 0px #0000001F',
+        light: '0rem 0.25rem 1.25rem 0rem #00000014',
+        medium: '0rem 0.25rem 1.25rem 0rem #0000001F',
       },
     },
     top: {
       big: {
-        light: '0px -4px 40px 0px #00000014',
-        medium: '0px -4px 40px 0px #0000001F',
+        light: '0.rem -0.25rem 2.25rem 0rem #00000014',
+        medium: '0rem -0.25rem 2.25rem 0rem #0000001F',
       },
       small: {
-        light: '0px -4px 20px 0px #00000014',
-        medium: '0px -4px 20px 0px #0000001F',
+        light: '0rem -0.25rem 1.25rem 0rem #00000014',
+        medium: '0rem -0.25rem 1.25rem 0rem #0000001F',
       },
     },
   },
   palette: newThemePalette,
   typography: {
+    body1: undefined,
+    body2: undefined,
+    bodyMdBold: {
```

Comment:
We never actually use any bold font-weights, so can you rename them to Semi-bold so its more consistent with the names in Figma?